source/PlayerInfo.cpp

```diff
@@ -4149,8 +4154,13 @@ void PlayerInfo::SortAvailable()
 					return false;
 			}
 			// Tiebreaker for equal PAY is ABC.
+			// Not using the Mission class' overridden comparison operator, as it orders by id, not display name.
 			case ABC:
 			{
+				if(lhs.Order() < rhs.Order())
+					return true;
+				else if(lhs.Order() > rhs.Order())
+					return false;
```

Comment:
Again, the `order` of a mission should have nothing to do with the sorting in the jobs board, only the order in which missions are offered.
```suggestion
			case ABC:
			{
```